src/styles/main.scss

```diff
@@ -1,3 +1,18 @@
-body {
-  margin: 0;
-}
```

Comment:
add styles for the body to a separate module for basic resets and basic element styling. It could improve clarity and clearness of your main.scss file